openquake/server/db/actions.py

```diff
@@ -116,21 +114,22 @@ def get_calc_id(datadir, job_id=None):
     calc_id = 0 if not calcs else calcs[-1]
     if job_id is None:
         try:
-            job_id = models.OqJob.objects.latest('id').id
-        except exceptions.ObjectDoesNotExist:
+            job_id = db('SELECT seq FROM sqlite_sequence WHERE name="job"',
```

Comment:
the old version  seemed to suggest that the result could be a list (getting the latest). Are we sure we obtain a single output here?